discovery.go

```diff
@@ -199,6 +199,12 @@ func (s *PHPStore) discoverPHPViaPHP(dir, binName string) *Version {
 		PHPPath:     php,
 	}
 	fpm := filepath.Join(dir, "sbin", strings.Replace(binName, "php", "php-fpm", 1))
+	fpmBin := filepath.Join(dir, "bin", strings.Replace(binName, "php", "php-fpm", 1))
```

Comment:
Instead, can you check if the file exists in `sbin` and switch to `bin` if it does not?

Reply:
changed :)